addons/purchase/i18n/es.po

```diff
@@ -2793,7 +2793,7 @@ msgstr "Método de redondeo del cálculo de impuestos"
 #. module: purchase
 #: model_terms:ir.ui.view,arch_db:purchase.purchase_order_form
 msgid "Tax excl."
-msgstr "Impuestos incluidos"
+msgstr "Impuestos no incluidos"
```

Comment:
This is usually translated as "Base imponible", but why not do this on Transifex?